content/docs/2019-06-23-introducing-mlrplayground.Rmd

```diff
@@ -0,0 +1,120 @@
+---
+title: "Introducing mlrPlayground"
+authors: ["sebastian-gruber"]
+date: "2019-06-23"
+output:
+  blogdown::html_page:
+    toc: true
+categories:
+  - r-bloggers
+tags:
+  - shiny
+  - visualization
+  - mlr
+  - teaching
+  - classification
+  - regression
+  - hyperparameter
+---
+
+<img src="https://i.imgur.com/MPEwCJ2.gif" width=800 />
+<br/>
+
+## First of all
+You may ask yourself how is this name ‘mlrPlayground’ even justified?
+What a madman puts two such opposite terms in a single word and expects people to take him seriously?
+I assume most of you know ‘mlr’, for those who don’t: it’s a framework for machine learning tasks offering a huge variety of different learners based on the best packages available in R, it’s about productivity, simplicity, and extendability.
+Quite the opposite from a place, where you can play with your best friends, make new friends, live out your fantasies and just have a great time the whole day until your parents pick you up.
+Well, for most of the readers here this may not be the case anymore – we know, we are still young in our heart, but let’s be honest … 
+
+For sure, we all have those memories and definitely have certain associations by the word ‘Playground’.
+So what is it then with this thing called ‘mlrPlayground’?
+<br/>
+
+## The idea
+The thought behind this project was to offer a platform in the form of a <a href="https://shiny.rstudio.com/">Shiny</a> web application, where a user can try out different kinds of learners provided by the mlr package on a small set of destinct and tunable regression and classification tasks, making it possible to observe the prediction/performance behaviour based on changes on the task, the learner or the learner's hyperparameters.
+In the best case the user will gain new insights and deeper understanding of how each learner works, where it’s advantages are and also very importantly the cases when the learner fails miserably.
+There are a lot of different settings we want to offer in the user interface, and so – to not remove the fun of our playground – a huge effort went into making the UI as aesthetically pleasing and as smooth as possible.
+To achieve this, a website template was downloaded from <a href="https://templated.co/">Templated</a> and used as a baseline for design. After extending the template with missing CSS classes, most of the used shiny widgets have been overwritten – or even completely replaced –, offering refreshingly new visuals for the well known shiny framework.
```

Comment:
Paragraph break before this sentence.

Always start sentences on a new line (for review purposes).

```suggestion
To achieve this, a website template was downloaded from <a href="https://templated.co/">Templated</a> and used as the baseline design. 
After extending the template with missing CSS classes, most of the used shiny widgets have been overwritten – or even completely replaced –, offering refreshingly new visuals for the well-known shiny framework.
```